docker/project.properties

```diff
@@ -8,9 +8,9 @@ resources.source.1.config.ansible-become=false
 resources.source.1.config.ansible-gather-facts=true
 resources.source.1.config.ansible-ignore-errors=true
 resources.source.1.config.ansible-inventory=/home/rundeck/data/inventory.ini
-resources.source.1.type=com.batix.rundeck.plugins.AnsibleResourceModelSourceFactory
-service.FileCopier.default.provider=com.batix.rundeck.plugins.AnsibleFileCopier
-service.NodeExecutor.default.provider=com.batix.rundeck.plugins.AnsibleNodeExecutor
+resources.source.1.type=com.rundeck.plugins.ansible.plugin.AnsibleResourceModelSourceFactory
+service.FileCopier.default.provider=com.rundeck.plugins.ansible.plugin.AnsibleFileCopier
+service.NodeExecutor.default.provider=com.rundeck.plugins.ansible.plugin.AnsibleNodeExecutor
```

Comment:
this hasn't changed, the name is set here 
https://github.com/rundeck-plugins/ansible-plugin/blob/enh/RUN-671_migrate_repo-220310/src/main/groovy/com/rundeck/plugins/ansible/plugin/AnsibleResourceModelSourceFactory.java#L17

https://github.com/rundeck-plugins/ansible-plugin/blob/enh/RUN-671_migrate_repo-220310/src/main/groovy/com/rundeck/plugins/ansible/plugin/AnsibleFileCopier.java#L27

https://github.com/rundeck-plugins/ansible-plugin/blob/enh/RUN-671_migrate_repo-220310/src/main/groovy/com/rundeck/plugins/ansible/plugin/AnsibleNodeExecutor.java#L25